software/github-actions/egg-github-actions.json

```diff
@@ -0,0 +1,72 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-12-24T15:03:37+01:00",
+    "name": "GitHub Actions",
+    "author": "raymond@reflexian.com",
+    "description": "Allows your to host your GitHub Self-Hosted Action runners on Pterodactyl",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": ".\/run.sh",
```

Comment:
I am not a huge fan of shell scripts as the startup for a server.

Reply:
The problem is that github runners are started using a script. 
So do you have a prefered way for this case ?

Reply:
Try to do what they do in that script in the startup cmd. Else you will have to pr it in to the needed image